doc/sphinx-guides/source/installation/config.rst

```diff
@@ -2391,6 +2391,9 @@ please find all known feature flags below. Any of these flags can be activated u
     * - api-session-auth
       - Enables API authentication via session cookie (JSESSIONID). **Caution: Enabling this feature flag exposes the installation to CSRF risks!** We expect this feature flag to be temporary (only used by frontend developers, see `#9063 <https://github.com/IQSS/dataverse/issues/9063>`_) and removed once support for bearer tokens has been implemented (see `#9229 <https://github.com/IQSS/dataverse/issues/9229>`_).
       - ``Off``
+    * - api-bearer-auth
+      - Enables API authentication via Bearer Token for OIDC User Accounts. **Information: This feature works only for ODIC UserAccounts!**
+      - ``Off``
```

Comment:
```suggestion
    * - api-bearer-auth
      - Enables API authentication via Bearer Token for OIDC User Accounts. **Information: This feature works only for OIDC UserAccounts!**
      - ``Off``
```

Small typo above ^^.

This is a great start on docs. They can be previewed here: https://dataverse-guide--9532.org.readthedocs.build/en/9532/installation/config.html

As I mentioned elsewhere, I pushed a commit with some additional suggested docs at 96e8e65. Please take a look.